platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -1026,7 +1026,8 @@ public PlatformStateAccessor getReadablePlatformState() {
      *
      * @param snapshotMetrics the metrics to set
      */
-    public void setSnapshotMetrics(MerkleRootSnapshotMetrics snapshotMetrics) {
+    @NonNull
```

Comment:
void methods don't get annotations - the needed nullity annotation I was referring to was on the parameter, `snapshotMetrics`

Reply:
oh sure, just updated